apps/passport-server/src/services/telegramService.ts

```diff
@@ -251,7 +339,10 @@ export class TelegramService {
     // hardcoded eventIDs and signing keys for SRW
     let signerMatch = false;
     let eventIdMatch = false;
-    if (process.env.PASSPORT_SERVER_URL === "http://localhost:3002") {
+    if (
+      process.env.PASSPORT_SERVER_URL === "http://localhost:3002" ||
+      process.env.PASSPORT_SERVER_URL === "https://dev.local:3002"
+    ) {
       eventIdMatch = true;
       signerMatch = true;
     } else if (process.env.PASSPORT_SERVER_URL?.includes("staging")) {
```

Comment:
we should update the logic around here to not be awful at some point.

Reply:
Resolved by 27edf975